### ISSUE TITLE
Catchable fatal error with some gateways

### ISSUE THREAD

**Issue by tamarazuk** [OnTheGoSystems/woocommerce-multilingual#1]
A mutual customer recently reported the following error when they updated to the latest version of WooCommerce Multilingual:

```
Catchable fatal error: Object of class WC_Gateway_Moneris_Credit_Card could not be converted to string in /.../wp-content/plugins/woocommerce-multilingual/inc/wc-strings.class.php on line 22
```

The array filtered via `woocommerce_payment_gateways` can have strings and class instances as elements. WooCommerce accomodates this two different types [here](https://github.com/woothemes/woocommerce/blob/f183df1b138b501a687c391681958d4f002c8ac9/includes/class-wc-payment-gateways.php#L104).

However, it looks like the change introduced in SHA 1ab3b5fa138022b448c1b2e06ad4cee2da7a4c1b does not take into account that `$gateway` can also be class instance on [line 22](https://github.com/OnTheGoSystems/woocommerce-multilingual/blob/ae95ed54a1535e3b631d2ff138d8bc92fd71ef69/inc/wc-strings.class.php#L22). This line should be changed to something like:

```
$gateway = is_string( $gateway ) ? new $gateway() : $gateway;
$gateway_id = $gateway->id;
```

Thanks!

**Comment by mihaigrigori** [comment]
Thanks for reporting this @tamarazuk 
We've addressed the issue and will include the fix in the following release shortly.

**Comment by mihaigrigori** [comment]
Fixed in version 3.7.9

**Comment by tamarazuk** [comment]
thanks for the fix @mihaigrigori :smile:
